trade-finance-manager-api/src/v1/controllers/deal.controller-test.js

```diff
@@ -0,0 +1,9 @@
+const MOCK_DEAL_MIA_SUBMITTED = require('../__mocks__/mock-deal-MIA-submitted');
+
+const dealController = require('./deal.controller');
+
+describe('dealCanBeSubmittedToACBS()', () => {
+  it('Should ascertain whether the deal can be submitted to the ACBS', async () => {
+    expect(dealController.dealCanBeSubmittedToACBS(MOCK_DEAL_MIA_SUBMITTED)).toEqual(false);
+  });
```

Comment:
Can we add a couple of tests for the true values as well - should be true when AIN or MIN?